File: JavaStuff/Reference/WhatsYourName.java
import java.util.Scanner;
public class WhatsYourName {
  public static void main(String args[]) {
    System.out.println("What's Your Name?");
    Scanner scanner = new Scanner(System.in);
    String Name = scanner.nextLine();
    System.out.println("Nice to Meet You, " + Name);
    scanner.close();
  }
}

File: Reference/c/whatsyourname.c
#include <stdio.h>
int main() {
    int foo = 8;
    char aString[] = "yeet";
    char firstName[100];
    printf("Type your first name: ");
    scanf("%s", &firstName);
    printf("Nice To Meet You, ");
    printf("%s", firstName);
    printf("!\n");
    return 0;
}